### PR TITLE
Code quality fix - Collapsible "if" statements should be merged.

### PR DESCRIPTION
**Pull request by faisal-hameed**
This pull request is focused on resolving occurrences of Sonar rule 
squid:S1066 - Collapsible "if" statements should be merged. 
You can find more information about the issue here: 
https://dev.eclipse.org/sonar/rules/show/squid:S1066

Please let me know if you have any questions.

Faisal Hameed

### CODE REVIEW

File: FileManager/src/org/openintents/filemanager/FileManagerActivity.java [hunk 1]
```diff
@@ -186,9 +186,8 @@ public boolean onOptionsItemSelected(MenuItem item) {
 	// The following methods should properly handle back button presses on every API Level.
 	@Override
 	public boolean onKeyUp(int keyCode, KeyEvent event) {
-		if (VERSION.SDK_INT > VERSION_CODES.DONUT) {
-			if (keyCode == KeyEvent.KEYCODE_BACK && mFragment.pressBack())
-				return true;
+		if (VERSION.SDK_INT > VERSION_CODES.DONUT && keyCode == KeyEvent.KEYCODE_BACK && mFragment.pressBack()) {
```
Comment (friedger) [inline FileManager/src/org/openintents/filemanager/FileManagerActivity.java]: I prefer the previous style to clearly state that this branch is about Android versions.

**Comment by faisal-hameed** [comment]
@friedger Reverted that branch about Android versions.

**Comment by friedger** [comment]
@faisal-hameed Could you rebase this?

**Comment by faisal-hameed** [comment]
@friedger Rebased.

**Pull request merged**